client/src/components/ProductDetails/Features.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Features ({currentProduct}) {
+  return (
+    <div>
+      {currentProduct.features.map((feature, idx) =>
+        // eslint-disable-next-line react/jsx-key
```

Comment:
I think you should be able to get rid of the eslint-disable...

Reply:
Forgot to delete, nice catch